analytics/analytics-ui/src/app/analytic/analytic.component.ts

```diff
@@ -23,15 +23,44 @@ import { AnalyticsService } from '../gaffer/analytics.service';
   templateUrl: './analytic.component.html'
 })
 @Injectable()
-export class AnalyticComponent implements OnInit {
+export class AnalyticComponent implements OnInit, AfterViewInit {
   @Input() model;
 
   constructor(
     private router: Router,
     private analyticsService: AnalyticsService
   ) { }
 
-  ngOnInit() { }
+  ngOnInit() {}
+
+  ngAfterViewInit() {
+
+    // Set the default icon if an icon is not specified
+    const defaultIcon = '<svg xmlns=\'http://www.w3.org/2000/svg\' width=\'24\' height=\'24\' viewBox=\'0 0 24 24\'>' +
```

Comment:
Which icon is this? Can we just store this in an svg file?

Reply:
This is the star icon.